src/ui/qml/NetworkSetupWizard.qml

```diff
@@ -93,43 +93,95 @@ ApplicationWindow {
         id: firstPage
 
         Column {
-            spacing: 8
+            spacing: 10
 
-            Label {
-                width: parent.width
-                text: qsTr("This computer's Internet connection is free of obstacles. I would like to connect directly to the Tor network.")
-                wrapMode: Text.Wrap
-                horizontalAlignment: Qt.AlignHCenter
-            }
+            Column {
+                width: 400
+                spacing: 5
 
-            Button {
-                anchors.horizontalCenter: parent.horizontalCenter
-                text: qsTr("Connect")
-                isDefault: true
-                onClicked: {
-                    // Reset to defaults and proceed to bootstrap page
-                    configPage.reset()
-                    configPage.save()
+                Image {
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    // Ideally we would use the svg here, but I cannot work out a way to get it to
+                    // display as nice as the png does.
```

Comment:
One useful hint is to use `sourceSize { width: n; height: n }` at the desired size for SVGs. That should make it render at the right resolution.